library/spdm_crypt_lib/libspdm_crypt_cert.c

```diff
@@ -1028,6 +1028,9 @@ static bool libspdm_verify_leaf_cert_spdm_extension(const uint8_t *cert, size_t
 /**
  * Certificate common Check for SPDM leaf cert when get_cert and set_cert.
  *
+ * For GenericCert certificate models,
```

Comment:
Can remove this. The original comment does not say what is or is not checked.

Reply:
Thanks. I have deleted the comment.